loris/resolver.py

```diff
@@ -25,13 +25,13 @@
 logger = getLogger(__name__)
 
 
-class _AbstractResolver(object):
+class _AbstractResolver:
 
     def __init__(self, config):
         self.config = config
         if config:
             self.auth_rules_ext = self.config.get('auth_rules_ext', 'rules.json')
-            self.use_extra_info = self.config.get('use_extra_info', True)
```

Comment:
I don’t know how many people have ever set this field in their config – it would be nice to check for the presence of this field, and drop a warning if so. Otherwise stuff will silently break when they update.

Something like:

```python
if "use_extra_info" in self.config and "use_auth_rules" in self.config:
    raise ConfigError("You cannot set both use_extra_info and use_auth_rules. Please remove use_extra_info from your config.")

if "use_extra_info" in self.config:
    warnings.warn("The use_extra_info field has been renamed to use_auth_rules and will be removed in a future version, please update your config", DeprecationWarning)
    self.config["use_auth_rules"] = self.config["use_extra_info"]
```

This might not be the place for it – maybe `config.py`?